libs/use-viewer/src/index.ts

```diff
@@ -182,14 +182,14 @@ const useViewer = ({ handleError, streamAccountId, streamName, subscriberToken }
     }
   };
 
-  const setSourceQuality = (sourceId: string, quality: SimulcastQuality) => {
+  const setSourceQuality = (sourceId: string, quality?: SimulcastQuality) => {
```

Comment:
So this function will update the stream quality and update the quality options UI? I strongly recommend separating the logic here. They should not be coupled together.  This function should only update quality. The UI should be depend on the quality state which get update in 'layer' event handler.

Reply:
This function updates stream quality only. Options UI is handled in the app based on `useViewer` state: https://github.com/dolbyio-samples/rts-app-react-publisher-viewer/blob/bc2f18629a91a03b78931de9441cf5fac7326aa3/libs/use-viewer/src/index.ts#L106-L110

Reply:
OK, so`UPDATE_SOURCE_QUALITY` is to update the current selection.